### PR TITLE
Update Now to Vercel.

### PR DESCRIPTION
**Pull request by leerob**
Hey! A few small changes. Could you also update the URL in the repo from `.now.sh` to `.vercel.app`? 🙏

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,83 +4,39 @@
...
 
-## Contents
+---
 
-- [Installation](#installation)
-- [Development Workflow](#development-workflow)
-- [Deploy to Now](#deploy-to-now)
-- [Deploy to Netlify](#deploy-to-netlify)
-- [Deploy to Gitpod](#deploy-to-gitpod)
+### Getting Started
```
Comment (leerob) [inline readme.md:9]: I copied this content from create-next-app, which is what all new Next.js projects start with.

File: vercel.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
 	"version": 2,
 	"name": "next-simple-starter",
-	"alias": "nextss.now.sh",
```
Comment (leerob) [inline vercel.json:4]: This isn't needed when you're using the GitHub integration and the Deploy button workflow 👍

File: readme.md (hunk 1 above)
Comment (ooade) [reply readme.md:9]: Cool. Thanks!

**Review by ooade — APPROVED** [review]

**Comment by ooade** [comment]
Thanks a lot @leerob 🙏

**Comment by leerob** [comment]
Thank you so much! This starter is awesome 😄

**Comment by leerob** [comment]
Oops, looks like nextss.vercel.app is taken. Could you update the link to next-ss.vercel.app instead? I confirmed that's working.

**Comment by ooade** [comment]
> Oops, looks like nextss.vercel.app is taken. Could you update the link to next-ss.vercel.app instead? I confirmed that's working.

Cool. I've updated it. Works fine now.

**Comment by ooade** [comment]
@leerob How was the alias assigned to this repo's deployment?

**Comment by leerob** [comment]
If you want it to sync back with this repo, then you should use the deploy button in the README to deploy this and then use that alias. I just deployed it myself and created an alias, but it'd be better if this repo was the source probably.

**Comment by ooade** [comment]
I've assigned nextss.vercel.app to the default branch 🙏

**Pull request merged**